### PR TITLE
Create HOC that accept style and className

### PR DESCRIPTION
**Pull request by WiNloSt**
Previously we have to export default component and another name export for component that accept `style` and `className` prop, because previous written HOC (Higher Order Component) will create another div and wrap it around wrapped component. This HOC, however, merge all props including `style` and `className` onto wrapped component. So it doesn't create an unwanted extra div.

Also, decorator syntax is somewhat not that widely use. We could discuss further whether or not we're going to use this syntax.

### CODE REVIEW

**Comment by prontonatty** [comment]
I read it. Is this the same thing that Net suggested for styling by merging props? Just want to confirm my understanding.

Anyway, good job!! And good night!!

**Comment by WiNloSt** [comment]
@prontonatty Yes

File: src/container/AcceptStyleAndClassName.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
 import React from 'react'
+import classNames from 'classnames'
+
+function isStateless (component) {
+  return !component.prototype.render
+}
```
Comment (lukyth) [inline src/container/AcceptStyleAndClassName.js]: I think we talked about stateless component before and we've concluded that we won't use stateless component. This checking might not really be necessary?

Comment (WiNloSt) [inline src/container/AcceptStyleAndClassName.js]: Right. Taking this pieces off will also make the code a bit more cleaner.

**Comment by WiNloSt** [comment]
Close this PR since we're gonna use branching instead. I'll create another PR when it's done.

**Pull request closed without merging**